tests/bsim/sh_common.source

```diff
@@ -50,18 +50,21 @@ function Execute() {
 }
 
 function _guess_test_relpath(){
-  local PA="$(realpath --relative-to "${ZEPHYR_BASE}" $(dirname "${BASH_SOURCE[2]}"))"
+  local PA="$(cd -- "$(dirname "${BASH_SOURCE[2]}")" && pwd)"
+  PA="$(realpath --relative-to "${ZEPHYR_BASE}" "${PA}")"
   echo $PA | sed -E 's/\/tests?_scripts//'
 }
 
 # For a caller running from a tests_script/ folder, get the path of its parent
 # relative to $ZEPHYR_BASE
+# This must be run without/before cd'ing into another directory
```

Comment:
what does that mean?

Reply:
That if you change directory in the calling script before calling this function, it will not work properly